packages/cw-orch-core/src/environment/mod.rs

```diff
@@ -16,4 +16,4 @@ pub use queriers::{
     DefaultQueriers, Querier, QuerierGetter, QueryHandler,
 };
 pub use state::{ChainState, StateInterface};
-pub use tx_handler::{TxHandler, TxResponse};
+pub use tx_handler::{AccessConfig, TxHandler, TxResponse};
```

Comment:
Are we sure about exposing this type? Don't think proto messages have good Rust API

Reply:
I wanted to expose it for users to be able to have it without having to import `cosmrs` themselves.
But having a better API with addresses as String instead of complicated types is also a good idea

Reply:
Does this mean that cosmrs is inside of cw-orch-core by default?

Reply:
nevermind, looks like cosmos-sdk-proto is already included there by default.

Reply:
New version is out with a slightly better API, what do you think ?